index.html

```diff
@@ -106,166 +93,151 @@ <h4 id="title-contact">
-                fill="#505F79" />
-            </svg>
-                    </a>
-                </li>
-            </ul>
-            <div class="card-cta-wrapper">
-                <button class="card-btn">Get my resume</button>
+            <div id="about-content">
+                <h2 id="about-title">About <br />Myself</h2>
+                <p id="about-description">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                    experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                
+                <h4 id="title-contact">
+                    LET’S CONNECT
+                </h4>
+                <ul class="icons-wrapper">
+                    <li>
+                        <a href="/">
+                            <img src="./assets/icons/twitter.svg" alt="Twitter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="/">
+                            <img src="./assets/icons/linkedin-icon.svg" alt="Linkedin" />
+                        </a>
+                    </li>
+                    <li>
+                        <a href="/">
+                            <img src="./assets/icons/m-icon.svg" alt="M">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="">
+                            <img src="./assets/icons/github.svg" alt="Github">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="/">
+                            <img src="./assets/icons/share.svg" alt="Share">
+                        </a>
```

Comment:
- [ ] [OPTIONAL] If you add a descriptive link to `<a href="/">` it would be better 👍🏼